ux/designs/monitormetrics/monitormetrics.md

```diff
@@ -0,0 +1,26 @@
+# Monitor: System Metrics
+This design covers the following workflows:
+
+- System wide metrics via the Home page
+
+
+## System Metrics
+
+![Image of choosing connection](img/dashboard.png)
+
+1. System wide metrics will be shown on the Home page of the application.
+1. Status aggregate cards will be used to show high level metrics.
+  - **Integrations**: Card shows number of total active integrations. Option to break down integrations via success vs. error.
+  - **Connections**: Shows total number of available connections. Optional - click on text to drill down to connection page.
+  - **Messages**: Show total number of messages. Option to break down via success vs error.
+  - **Uptime**: Shows time that current pod is running since last restart
```

Comment:
I think in the 'uptime' card there should be no time series graph as we do not record actually time series data (this could be added later if we go with the prometheus approach)

Reply:
@rhuss Makes sense. We can remove the time series graph. Keep the rest of the content? Such as the date/time started and length of time.